langfuse_obs.py

```
@@ -360,3 +360,92 @@ def emit_chat_observation(
+            email=f"{actor}@agency.internal",
+            department=operation,
+            key_id=None,
+            model=model or "agency-internal",
+            messages=[{"role": "user", "content": safe_input or "(no input)"}],
+            output_text=safe_output or "(no output)",
+            prompt_tokens=0,
+            completion_tokens=0,
+            meta=meta,
+            task_name=f"agency:{operation}",
+        )
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)